README.md

```diff
@@ -109,9 +109,30 @@ will need to set it up with these steps:
 Check [CONTRIBUTING.md](https://github.com/chshersh/iris/blob/main/CONTRIBUTING.md)
 for contributing guidelines.
 
-To build the project and run the tests, use `cabal`:
+## Building the project
+
+To build the project and run the tests locally, you can use either `cabal` or `stack`:
 
 ```shell
 cabal build all
-cabal test --enable-tests --test-show-details=direct
+cabal test --enable-test --test-show-details=direct
```

Comment:
Looks like a typo and it should be `--enabled-tests` and not `--enable-test` according the the `cabal test --help`:

```
$ cabal test --help
...
 --enable-tests                 Enable dependency checking and compilation for
                                test suites listed in the package description
                                file.
```

```suggestion
cabal test --enable-tests --test-show-details=direct
```

Reply:
Yep, you are right. Done!